src/overseer/util.clj

```diff
@@ -0,0 +1,8 @@
+(ns overseer.util)
+
+(defn when-update
+  "Invoke (apply update m k f args) only if m contains k"
+  [m k f & args]
+  (if (contains? m k)
+    (apply update m k f args)
+    m))
```

Comment:
Potentially a good fit for a std function. If so, this ns is to be deleted once that goes out.